frappe/types/df.py

```diff
@@ -0,0 +1,37 @@
+"""This file defines all frappe types."""
+
+from datetime import date, datetime, time
+
+# DocField types
+Data = str
+Text = str
+Autocomplete = Data
```

Comment:
Making our own types so we can extend in future. Tried TypeAlias but it doesn't work well with pyright (forgets underlying type in containers)... will give it a shot later.

Reply:
`Select = Literal` also doesn't work. Using literal for now.